cmd/kube-controller-manager/app/plugins_storageless.go

```diff
@@ -0,0 +1,156 @@
+// +build storageless
+
+/*
+Copyright 2014 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package app
+
+import (
+	// This file exists to force the desired plugin implementations to be linked.
+	// This should probably be part of some configuration fed into the build for a
+	// given binary target.
+
+	"fmt"
+	"k8s.io/klog/v2"
+
+	// Cloud providers
+	cloudprovider "k8s.io/cloud-provider"
+	// ensure the cloud providers are installed
+	_ "k8s.io/kubernetes/pkg/cloudprovider/providers"
+	// Volume plugins
+	"k8s.io/kubernetes/pkg/volume"
+	"k8s.io/kubernetes/pkg/volume/csi"
+	"k8s.io/kubernetes/pkg/volume/fc"
+	"k8s.io/kubernetes/pkg/volume/flexvolume"
+	"k8s.io/kubernetes/pkg/volume/hostpath"
+	"k8s.io/kubernetes/pkg/volume/iscsi"
+	"k8s.io/kubernetes/pkg/volume/local"
+	"k8s.io/kubernetes/pkg/volume/nfs"
+	"k8s.io/kubernetes/pkg/volume/rbd"
```

Comment:
Should it be named "intree_plugins" instead of "storageless"?  Volume plugins such as csi, fc, iscsi, rbd, etc. still have external storage associated.

Reply:
Also plugins such as RBD will be migrated to CSI as well.